### ISSUE TITLE
does it support detecting multiple categories at the same time?

### ISSUE THREAD

**Issue by smartkyx** [tianrun-chen/SAM-Adapter-PyTorch#130]

**Comment by smartkyx** [comment]
The provided dataset seems to contain only one category. Does it support detecting multiple categories at the same time? Also, I noticed that the input only requires an image and a mask, and the JSON file doesn’t seem to play any role.

**Comment by SHITINGCHEN** [comment]
> The provided dataset seems to contain only one category. Does it support detecting multiple categories at the same time? Also, I noticed that the input only requires an image and a mask, and the JSON file doesn’t seem to play any role.

I have the same question. Author of MedSAM3 said they finetuned SAM3 using "paired medical images and concise concept phrases". I am wondering if text label should also be used in fine-tuning.
